typed/merge.go

```diff
@@ -180,14 +180,18 @@ func (w *mergingWalker) visitListItems(t *schema.List, lhs, rhs value.List) (err
 	}
```

Comment:
AI: look for issue reported where someone couldn't update because of managed fields